### PR TITLE
Redirect cyclist to their latest assigned pickups

### PR DESCRIPTION
**Pull request by seanpdoyle**
**Extract `Session` model for login form**

Create the `Session` ActiveModel::Model to cleanup and more easily
internationalize the login form.

**Redirect cyclist to their latest assigned pickups**

https://trello.com/c/wvpMTQHG

When cyclists volunteer, administrators will create account for them.

In the future, when there are more active zones, administrators will
assign cyclists to zones.

For the time being, any cyclist account that is created will be assigned
to the first created zone (which is currently `80205` in `staging` and
`production`).

This commit introduces the plumbing to assign a cyclist to a single
zone, maximizing future flexibility.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -16,6 +17,22 @@ class User < ActiveRecord::Base
     message: I18n.t("validations.accepted"),
   }
 
+  def cyclist=(flag)
```
Comment (geoffharcourt) [inline app/models/user.rb]: I'm a little confused by the name `flag` here.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,19 @@ After setting up, you can run the application using [Heroku Local]:
...
+  <dt>An administrator</dt>
+  <dd>admin@example.com</dt>
+
+  <dt>A donor</dt>
+  <dd>donor@example.com</dt>
+
+  <dt>A cyclist</dt>
+  <dd>cyclist@example.com</dt>
+</dl>
+
```
Comment (geoffharcourt) [inline README.md]: 👍

**Comment by geoffharcourt** [comment]
One question about naming, but LGTM.

File: app/models/user.rb (hunk 1 above)
Comment (seanpdoyle) [inline app/models/user.rb]: Would `is_cyclist` an improvement?

**Pull request merged**